python/py-scipy/Portfile

```diff
@@ -93,19 +102,14 @@ if {${name} ne ${subport}} {
         # Requires threads.h, missing in Xcode clang
         # https://build.macports.org/builders/ports-10.11_x86_64-builder/builds/136856/steps/install-port/logs/stdio
         compiler.blacklist-append {clang < 900}
-
     }
 
     depends_lib-append      port:py${python.version}-numpy \
-                            port:py${python.version}-nose \
                             port:py${python.version}-pybind11
 
     depends_build-append    port:swig-python \
                             port:py${python.version}-pip
 
-    depends_test-append     port:py${python.version}-py \
```

Comment:
Same per the first comment

Reply:
that is a library dependency of `pytest`, so there isn't a reason I can think of to list it separately here